src/index.js

```diff
@@ -96,7 +96,7 @@ export async function renderHandlebarsTemplate(files, outputDirectory = process.
       await writeFile(path, htmlContents, 'utf8');
     }
     debug(`Wrote ${path}`);
-    console.err(`Wrote ${path} from ${file}`);
+    console.error(`Wrote ${path} from ${file}`);
```

Comment:
@keithamus I think we should not log ${path} when we are using stdout.
It gives a misleading output